Mathlib/Algebra/DirectLimit.lean

```diff
@@ -167,6 +167,15 @@ theorem lift_unique [Nonempty ι] [IsDirected ι (· ≤ ·)] (F : DirectLimit G
   DirectLimit.induction_on x fun i x => by rw [lift_of]; rfl
 #align module.direct_limit.lift_unique Module.DirectLimit.lift_unique
 
+lemma lift_injective [Nonempty ι] [IsDirected ι (· ≤ ·)]
+    (injective : ∀ i, Function.Injective <| g i) :
+    Function.Injective $ lift R ι G f g Hg := by
+  simp_rw [← LinearMap.ker_eq_bot, Submodule.eq_bot_iff, LinearMap.mem_ker] at injective ⊢
```

Comment:
I'm introducing some Unique instances to remove Nonempty hypothesis [here](https://github.com/leanprover-community/mathlib4/commit/089e987cd69b1854af99c1aa2ca75593850f5aaa). Also, `injective_iff_map_eq_zero` does your three rewrites in one step.

Reply:
Thanks! Can you merge the removal of nonempty assumption. i think the nonempty assumption in`lift_unique` can also be removed.

Reply:
Removed from `lift_unique` too